pkgs/applications/blockchains/groestlcoin/default.nix

```
@@ -51,12 +52,28 @@ stdenv.mkDerivation rec {
     ++ lib.optionals withWallet [ db53 sqlite ]
     ++ lib.optionals withGui [ qrencode qtbase qttools ];
 
-  postInstall = lib.optionalString withGui ''
+  postInstall = ''
+    installShellCompletion --bash contrib/completions/bash/groestlcoin-cli.bash
+    installShellCompletion --bash contrib/completions/bash/groestlcoind.bash
+    installShellCompletion --bash contrib/completions/bash/groestlcoin-tx.bash
+
+    installShellCompletion --fish contrib/completions/fish/groestlcoin-cli.fish
+    installShellCompletion --fish contrib/completions/fish/groestlcoind.fish
+    installShellCompletion --fish contrib/completions/fish/groestlcoin-tx.fish
+    installShellCompletion --fish contrib/completions/fish/groestlcoin-util.fish
+    installShellCompletion --fish contrib/completions/fish/groestlcoin-wallet.fish
```

```suggestion
    for file in contrib/completions/fish/groestlcoin-*.fish; do
      installShellCompletion --fish $file
    done
```
we can shorten this a bit

---

Done